Session6/Polynomial_Operations/PolyTest.java

```diff
@@ -0,0 +1,93 @@
+package Polynomial_Operations;
+
+import static org.junit.Assert.*;
+
+import java.lang.Character.Subset;
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+
+@RunWith(Parameterized.class)
+public class PolyTest {

```

Comment:
Update this test class with proper test methods

Reply:
Sir , please explain this ?

Reply:
Include Negative and positive test case both.